### PR TITLE
new koji build metadata osbs_build providing information about build

### PR DESCRIPTION
**Pull request by rcerven**
kind, subtypes and engine

* OSBS-8618

Signed-off-by: Robert Cerven <rcerven@redhat.com>

Maintainers will complete the following section:
- [x] Commit messages are descriptive enough
- [x] "Signed-off-by:" line is present in each commit
- [x] Code coverage from testing does not decrease and new code is covered
- [x] JSON/YAML configuration changes are updated in the relevant schema
- [x] Changes to metadata also update the documentation for the metadata
- [x] Pull request includes link to an osbs-docs PR for user documentation updates
- [x] New feature can be disabled from a configuration file

### CODE REVIEW

File: atomic_reactor/plugins/exit_koji_import.py [hunk 1]
```diff
@@ -425,10 +429,16 @@ def get_build(self, metadata, worker_metadatas):
...
+            extra['osbs_build']['kind'] = KOJI_KIND_IMAGE_BUILD
+            extra['osbs_build']['engine'] = self.workflow.builder.tasker.build_method
+            if 'operator_manifests' in extra:
+                extra['osbs_build']['subtypes'].append(KOJI_SUBTYPE_OP_APPREGISTRY)
         else:
             source_result = self.workflow.prebuild_results[PLUGIN_FETCH_SOURCES_KEY]
             extra['image']['sources_for_nvr'] = source_result['sources_for_nvr']
             extra['image']['sources_signing_intent'] = source_result['signing_intent']
+            extra['osbs_build']['kind'] = KOJI_KIND_IMAGE_SOURCE_BUILD
+            extra['osbs_build']['engine'] = 'bsi'
```
Comment (mlangsdorf) [inline atomic_reactor/plugins/exit_koji_import.py]: 'bsi' should probably be a constant here and the unit test.

Comment (rcerven) [reply atomic_reactor/plugins/exit_koji_import.py]: updated

**Comment by MartinBasti** [comment]
New attrs should be docuemnted

**Comment by MartinBasti** [comment]
otherwise LGTM

**Comment by rcerven** [comment]
> New attrs should be docuemnted

added to koji.md

**Comment by rcerven** [comment]
rn updated: new koji build metadata osbs_build providing information about build

**Pull request merged**